### PR TITLE
Ft dob

### PR DESCRIPTION
**Pull request by Porim**
Date of birth page and some tests. Not sure what more to say about it. Perhaps I might add some error messages later on that helps to write the correct date, but it should be enough for now. Actually, it already includes an automatic fulfilling in case a user inserted his DOB in his browser before.

### CODE REVIEW

**Comment by Joseaie** [comment]
Could you add screenshots of the pages please?

File: cypress/e2e/spec.cy.js [hunk 1]
```diff
@@ -17,3 +17,16 @@ describe('landing page', () => {
     cy.url().should('include', '/my-council-account')
   })
 })
+
+describe('date of birth', () => {
+  it('displays dob page', () => {
+    cy.visit('http://localhost:3000/date-of-birth.html')
+    cy.contains('What is your date of birth?')
+    cy.contains('Month')
+  })
```
Comment (Joseaie) [inline cypress/e2e/spec.cy.js:27]: nit: please add a new line

File: public/date-of-birth.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  </footer>
+
+  <script src="/javascripts/gov.js"></script>
+  <script>
+    window.GOVUKFrontend.initAll()
+
+  </script>
+</body>
+
+</html>
```
Comment (Joseaie) [inline public/date-of-birth.html]: nit: please add a new line to the end of the file

**Comment by Porim** [comment]
What about tests? Keep them in spec.cy.js file or move them to a separate file?

**Comment by ssrome** [comment]
Not a big thing now but can move tests into their own file, going forward.

File: public/date-of-birth.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      
+        </fieldset>
+      </div>
+    </main>
+    <div class="govuk-button-group">
+      <button data-prevent-double-click="true" class="govuk-button" data-module="govuk-button" onclick="document.location='email-address.html'">
+        Continue
+      </button>
+    
+      <a class="govuk-link" href="index.html">Cancel</a>
```
Comment (ssrome) [inline public/date-of-birth.html]: Just wondering about the cancel link as we don't seem to have them on any of the pages

**Review by ssrome — APPROVED** [review]

**Review by J-Coke — APPROVED** [review]

**Pull request merged**